app/javascript/components/table/Table.vue

```diff
@@ -62,9 +62,45 @@
 
       // open/close expandable section
       eventHub.$on('toggleRow', this.toggleRow)
+
+      //check url for theme parameter and add it to the selected filter options
+      this.checkThemeParameter()
     },
 
     methods: {
+      checkThemeParameter () {
+        let queryString = window.location.search.slice(1);
+        
+        if(queryString) {
+          let obj = {}
+
+          // anything after # is not part of query string, so get rid of it
+          queryString = queryString.split('#')[0]
+
+          // split the query string into its component parts
+          let arr = queryString.split('&')
+
+          for (let i=0; i<arr.length; i++) {
+
+            // separate the keys and the values
+            const a = arr[i].split('=');
+
+            // build up object
+            obj[a[0]] = a[1]
+          }
```

Comment:
Are there any cases for which we might have more than one parameter?
If this `checkThemeParameter` function is called just when clicking on one of the themes, wouldn't it be easier to just do something like `queryString.split('=')[1]` to get the selected theme?

I might be missing something.

Reply:
I was just wondering. It might not make perfect sense now, but it works well if we change something in the future. So let's keep it :)